docs/scripts/formattedTSDemos.js

```diff
@@ -42,7 +42,7 @@ const babelConfig = {
 
 const workspaceRoot = path.join(__dirname, '../../');
 
-async function getFiles(root) {
+async function getFiles(root, excludeRoot = false) {
```

Comment:
I don't think we want to transpile files at the root of `docs/data`, e.g. https://github.com/mui/mui-x/blob/master/docs/data/chartsApiPages.ts